### PR TITLE
test: fix disabled tests

### PR DESCRIPTION
**Pull request by I-vasilich-I**
**🟢 Add `deploy` label if you want to deploy this Pull Request to staging environment**

#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Do not put WIP in title. Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality
- Consider to add `area:*` label(s)

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [x] Test Case
- [ ] Documentation update
- [ ] Other

#### 💡 Background and solution

fix disabled tests

#### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Database migration is added or not needed
- [x] Documentation is updated/provided or not needed
- [x] Changes are tested locally

### CODE REVIEW

File: client/src/modules/Opportunities/components/NoConsentView/index.test.tsx [hunk 1]
```diff
@@ -29,66 +29,125 @@ describe('NoConsentView', () => {
...
-    // Check that we have all visible texts
     expect(modalTitle).toBeInTheDocument();
-    confirmationModalInfo.en.availableDataList.forEach(text => expect(screen.getByText(text)).toBeInTheDocument());
 
-    // Check that title tooltip works correctly
-    const titleTooltipIcon = await screen.findByTestId(confirmationModalInfo.ru.header);
+    // close modal
+    fireEvent.click(screen.getByRole('button', { name: 'Cancel' }));
+
+    await waitFor(() => {
```
Comment (aaliakseyenka) [inline client/src/modules/Opportunities/components/NoConsentView/index.test.tsx:37]: curious won't it be sufficient to `act` the click and then check the document. as of waiter would just make the test slower

Comment (I-vasilich-I) [reply client/src/modules/Opportunities/components/NoConsentView/index.test.tsx:37]: 1. image
2. It brakes tests even if we ignore eslint

Comment (I-vasilich-I) [reply client/src/modules/Opportunities/components/NoConsentView/index.test.tsx:37]: image
still brakes tests, I guess we need to wait for modal to close

Comment (I-vasilich-I) [reply client/src/modules/Opportunities/components/NoConsentView/index.test.tsx:37]: there's something ain't right on the test: if I won't close modal on any test, it brakes next one, modal stays in the DOM for some reason, even if I use `cleanup` in `beforeEach`. I managed to fix it only with `waitFor`.

Comment (aaliakseyenka) [reply client/src/modules/Opportunities/components/NoConsentView/index.test.tsx:37]: got it sorry didn't catch the problem initially.

**Review by aaliakseyenka — APPROVED** [review]

**Pull request merged**